api/metrics_service.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "google/api/annotations.proto";
+import "metrics.proto";
+
+//Service to fetch metrics from Proxy. This uses Promotheus data model to represent metrics returned.
+service MetricsService {
+  rpc FetchMetrics (MetricsRequest) returns (MetricsResponse) {
+    option (google.api.http) = {
+      post: "/v2/metrics"
```

Comment:
FYI, unlike the other services in `data-plane-api`, this is the first to be served by Envoy rather than have Envoy act as client. I'm wondering if we somehow want to distinguish this namespace wise. Also, can you clarify if this is intended to be a gRPC service, REST or combination?

Reply:
I do not have strong opinion on putting it in different name space. I was originally thinking of it as a gRPC service. For REST, I think the admin end point with ?format=json is sufficient I guess.